assignment/src/com/assignmentweek4/springwithxml/SetterDemoApp.java

```diff
@@ -0,0 +1,23 @@
+package com.assignmentweek4.springwithxml;
+
+import org.springframework.context.support.ClassPathXmlApplicationContext;
+
+public class SetterDemoApp {
+
+	public static void main(String[] args) {
+		// TODO Auto-generated method stub
+		try {
+		ClassPathXmlApplicationContext context =
+				new ClassPathXmlApplicationContext("applicationContext.xml");
```

Comment:
Use a String constant for "applicationContext.xml".

Reply:
Define one String variable for "applicationContext.xml" and use that variable inside new ClassPathXmlApplicationContext(variableName);